kebechet/managers/README.rst

```diff
@@ -36,5 +36,22 @@ If you wish to operate on repository source code, you can request to clone it:
 
 The last thing you need to do, is to register your manager to `REGISTERED_MANAGERS` constant (you can find it in `kebechet/managers/__init__.py` file) so Kebechet knows about your manager. Manager can be referenced by its name in lowercase (class name without the "manager" suffix).
 
+Overlays
+========
+
+Thoth allows users to specify overlays consisting of different hardware and software environments.  These environments
+are specified in a users .thoth.yaml file, files associated with a specific overlay are located in
+`<overlays-dir>/<environment-name>`.  Thoth will specify overlays to act on, if you create a manager which acts on a
+specific overlay then the desired behaviour is as follows.
+
+* if no overlays directory is specified in .thoth.yaml and no environment is passed, then the manager should act only on
+the first entry in `environments` in .thoth.yaml and changes should be made to the top level directory
+* if an overlays directory is specified and no environment is passed the manager should act on every overlay and make
```

Comment:
what you mean no environment is passed? I think runtime_environment is mandatory key in .thoth.yaml file, so it should have at least one right? cc @fridex

Reply:
Yes, the key is mandatory, we do not require any entries in it though (can be an empty list). This is something we could improve on thamos side.

Reply:
Here I am talking about whether `runtime_environment` was passed to `kebechet`